extensions/oauth/src/main/java/org/mule/extension/oauth2/internal/tokenmanager/TokenManagerConfig.java

```diff
@@ -23,15 +30,34 @@
  *
  * It can be referenced to access the state inside a flow for custom processing of oauth dance content.
  */
+@Alias("token-manager-config")
+@XmlHints(allowTopLevelDefinition = true)
 public class TokenManagerConfig implements Initialisable, MuleContextAware {
```

Comment:
I tried doing that, but i stumbled upon an issue similar to the one where the `@Inject` for mule context doesn't work.
In this case, the `@UseConfig` annotation is ignored, so i cannot get the config from the HttpAuthentication implementation. Will create a JIRA to do this after this other issue is fixed.